### PR TITLE
Added missing null checks

### PR DESCRIPTION
**Pull request by kasthack**

### CODE REVIEW

**Comment by Pilchie** [comment]
:-1: There are no circumstances when those values should ever be null.

**Comment by Pilchie** [comment]
Tagging @dotnet/roslyn-ide  for other reviewers.

File: src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs [hunk 1]
```diff
@@ -98,7 +98,7 @@ protected override Task ProduceTagsAsync(TaggerContext<IClassificationTag> conte
             if (_classificationService == null)
             {
                 var document = spanToTag.Document;
-                _classificationService = document?.Project.LanguageServices.GetService<IEditorClassificationService>();
+                _classificationService = document?.Project?.LanguageServices?.GetService<IEditorClassificationService>();
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs]: If you have a document, then it must have a project, so Project can't be null.  Same with LanguageServices.  The .?s here should be unnecessary.

**Comment by jmarolf** [comment]
@kasthack thanks for the contribution! On roslyn there are a set of things *that should never happen* :) and one of those is a document with a null project.  It would be better for us to crash and find out immediately that this has happened than to continue on regardless.  There are still plenty of [up for grabs issues](https://github.com/dotnet/roslyn/issues?q=is%3Aopen+is%3Aissue+label%3A%22Up+for+Grabs%22) that you can take a look at.  Please don't hesitate to ask about any of them.

**Pull request closed without merging**